zkevm-circuits/src/mpt_circuit.rs

```diff
@@ -424,7 +424,7 @@ impl<F: Field> MPTConfig<F> {
                         let rlp_value = self.rlp_item.assign(
                             &mut cached_region,
                             offset + idx,
-                            bytes,
+                            bytes, // highlight for comment
                             *item_type,
                         )?;
```

Comment:
The `bytes` passed here is `Hex` struct and the assign takes `&[u8]` for this argument.
The compiler is happy about this because we've implement the Deref trait for Hex. The `bytes` here is a Vec<u8> dereffed from the Hex struct.

Reply:
Wonderful!